src/main/java/com/redhat/quarkus/mandrel/collector/report/endpoints/ImageStatsResource.java

```diff
@@ -79,6 +83,63 @@ public ImageStats[] getByTag(@PathParam("tag") String tag) {
         return collection.getAllByTag(tag);
     }
 
+    @RolesAllowed("token_read")
+    @GET
+    @Path("experiment/{image_name}")
+    public Response getExperiment(@PathParam("image_name") String imageName) {
+        final ImageStats[] s = collection.getAllByImageName(imageName);
```

Comment:
We could consider exposing this as `@Path("name/{image_name}"` as well. Seems useful. But, another PR :)